templates/blog/article.html

```diff
@@ -21,127 +26,140 @@
 
 {% block extra_metatags %}
 
-<script type="application/ld+json">
-  {
-    "@context": "http://schema.org",
-    "@id": "https://canonical.com/#article",
-    "@type": "Article",
-    "name": "{{ article.title.renderered|safe }}",
-    "headline": "{{ article.excerpt.raw }}",
-    "author": {
-       "@type": "Person",
-       "name": "{{ article.author.name }}"
-    },
-    "datePublished": "{{ article.date_gmt }}",
-    {% if article.image and article.image.source_url %}
+  <script type="application/ld+json">
+    {
+      "@context": "http://schema.org",
+      "@id": "https://canonical.com/#article",
+      "@type": "Article",
+      "name": "{{ article.title.renderered|safe }}",
+      "headline": "{{ article.excerpt.raw }}",
+      "author": {
+        "@type": "Person",
+        "name": "{{ article.author.name }}"
+      },
+      "datePublished": "{{ article.date_gmt }}",
+      {
+        %
+        if article.image and article.image.source_url %
+      }
       "image": "{{ article.image.source_url }}",
-    {% endif %}
-    "url": "{{ request.url }}",
-    "publisher": {
-      "@type": "Organization",
-      "name": "Ubuntu"
+      {
+        %
+        endif %
+      }
```

Comment:
@petesfrench and @akbarkz these template logic blocks are being rendered into the source of the live website. Causing many errors to be reported by Google's search bot. Could you fix this please?